frontend/src/types.ts

```diff
@@ -600,3 +600,26 @@ export enum EventStatus {
   IN_PROGRESS = 'In Progress',
   ERROR = 'Error',
 }
+export type GroupsConfig = {
+  adminGroups: GroupStatus[];
+  allowedGroups: GroupStatus[];
+  errorAdmin?: string;
+  errorUser?: string;
+};
+
+export enum GroupsConfigField {
+  ADMIN,
+  USER,
+}
+
+export type GroupStatus = {
+  id: number;
+  name: string;
+  enabled: boolean;
+};
+
+export type MenuItemStatus = {
+  id: number;
+  name: string;
+  enabled: boolean;
+};
```

Comment:
Move these types into your page folder. They are not global concepts and we should look to start breaking up what is and isn't a global concept as we grow.

Reply:
yeah, i totally agree on that, I put them here as I was following the trend, but they should be totally in my component file.